sinch/domains/voice/models/callouts/requests.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass
+from typing import Literal
+from sinch.core.models.base_model import SinchRequestBaseModel
+from sinch.domains.voice.models import Destination, ConferenceDTMFOptions
+
+
+@dataclass
+class TextToSpeechVoiceCalloutRequest(SinchRequestBaseModel):
+    destination: Destination
+    cli: str
+    dtmf: str
+    domain: str
```

Comment:
It would be best to apply the same update as for `ConferenceVoiceCalloutRequest.domain`

Reply:
👍